beekeeper-api/README.md

```diff
@@ -0,0 +1,3 @@
+## Beekeeper API
+
+In progress :) !
```

Comment:
I would put better a short description of what is going to do with Beekeeper API and what you are pushing right now. You can say that it is in progress.